schema.py

```diff
@@ -0,0 +1,43 @@
+import graphene
+
+from graphene import relay, Schema
+from graphene_sqlalchemy import SQLAlchemyObjectType, SQLAlchemyConnectionField
+from models import User, Story, db_session
+
+
+class UserType(SQLAlchemyObjectType):
+    class Meta:
+        model = User
+        interfaces = (relay.Node, )
+
+
+class StoryType(SQLAlchemyObjectType):
+    class Meta:
+        model = Story
+        interfaces = (relay.Node, )
+
+
+class CreateStory(graphene.Mutation):
+    class Arguments:
+        story = graphene.String()
+        user_id = graphene.Int()
+    story = graphene.Field(StoryType)
+
+    def mutate(self, info, story, user_id):
```

Comment:
Having fields passed to the mutate class as positional arguments is a bad practice as it kills flexibility, it restricts as to always supply the fields not regarding whether the fields are optional at the model level, consider using kwargs and  looping through them to only save supplied fields

Reply:
Point taken, let me work on this. Thanks